src/simplnx/Utilities/ImageRotationUtilities.cpp

```diff
@@ -242,7 +242,15 @@ ImageRotationUtilities::Matrix4fR GenerateScaleTransformationMatrix(const Vector
   return transformationMatrix;
 }
 
-//------------------------------------------------------------------------------
+#define IR_SQUARE(value) ((value) * (value))
```

Comment:
```suggestion
template <class T>
constexpr T square(T value) noexcept
{
  return value * value;
}
```
We can pick a different name if we want.